### DISCUSSION TITLE
Running nodejs app on firecracker

### DISCUSSION THREAD

**Discussion by chenthillrulz** [nuta/kerla#164] [General]
I see there is a roadmap being planned for running nodejs app on firecracker. Wanted to check with you on your findings for the same ? Am working on a similar usecase and would be helpful if you can share your findings.

**Comment by nuta** [comment]
👋 Hi! I didn't quite get that but you mean you're working your own OS project to run Node.js on Firecracker?

**Comment by chenthillrulz** [comment]
I was trying to get details on https://github.com/nuta/kerla/projects/1 - Roadmap - Run a Node.js Web Application on Kerla on Firecracker on AWS

**Comment by nuta** [comment]
The roadmap is currently stale to focus on another OS project (
https://github.com/nuta/ftl). The new one might take over the road map as
its Linux compatibility layer.
